data/messages/templates/offerings/2023-07-pilot-confirmation.en.sms.liquid

```diff
@@ -0,0 +1 @@
+Thanks for your order! Click the following link to view your order {{ app_url }}/unclaimed-orders. Respond to this text if you need help. - suma
```

Comment:
There's no 'click' on a phone. 'Go to this link' or 'Follow this link' or 'Press this link'

Reply:
Thanks Rob, I made the [changes here](https://github.com/lithictech/suma/pull/414/commits/0971f43050c4b06602a0d910332dc8a1b3aef6ac). I also noticed a spelling mistake in the Spanish version that is fixed now.